docs/sites.yml

```
@@ -7846,3 +7846,12 @@
   built_by: Arturo Alviar
   built_by_url: "https://github.com/arturoalviar"
   featured: false
+- title: Pearly
+  url: https://www.pearlyplan.com
+  main_url: https://www.pearlyplan.com
+  description: >
+    Dental Membership Growth Platform
+  categories:
+    - Technology, Healthcare, App
```

```suggestion
    - Technology
    - Healthcare
    - App
```

Each category belongs on an individual line but otherwise looks great!